packages/hooks/README.md

```diff
@@ -164,4 +164,4 @@ const Example = ({nextValue: number}) => {
 
 # useIsomorphicLayoutEffect
 
-Drop-in replacement for `useLayoutEffect` that does not produce warnings during server-side rendering
+Drop-in replacement for `useLayoutEffect` that does not produce warnings during server-side rendering. If your project uses the `react-hooks/exhaustive-deps` linter rule, it's recommended to add `useIsomorphicLayoutEffect` to the `additionalHooks` of that rule.
```

Comment:
Including a full example may help, as I'm sure you'll need to redirect the user to our example or the docs page.